### ISSUE TITLE
command line flag to automatically open browser

### ISSUE THREAD

**Issue by thescientist13** [lwsjs/local-web-server#97]
Not sure if I missed it in the CLI docs, but is there a flag that would automatically open the browser to the running webserver?

**Comment by 75lb** [comment]
no, that's not currently a feature and not desirable as

a) The server application you're developing might not have a front-end to view in the browser (e.g. an HTTP API or service)
b) You might already have the browser open and not want a new tab each time to restart the server.

In iTerm you can control+click one of the "serving on" URLs to open the page in the browser.. there are many ways you can open the browser from the terminal, you can google yourself.

Let me know if there's anything else! 👍

**Comment by thescientist13** [comment]
For sure, I understand the case of not making it the _default_ option which is totally reasonable, but not even a flag _to_ enable it from the CLI if a user wanted?   Just something like:
```shell
ws --open
``` 

I think adding this feature would be appreciated by frontend developers like myself, as well as not having to assume everyone would have access to iTerm / OSX.  It would also help establish feature parity with other contemporaries in the community.  (fwiw, I found this repo through a comment over at [**http-server**](https://github.com/indexzero/http-server/issues/306#issuecomment-315072898))

Would you be open to it as a PR?  I would be happy to help if you would be open to pointing me in the right direction!

Thanks, I've been enjoying using this!

**Comment by thescientist13** [comment]
Thanks, I'll start taking a look into this!

**Comment by thescientist13** [comment]
[PR opened](https://github.com/lwsjs/lws/pull/9) @75lb

**Comment by thescientist13** [comment]
@75lb 
Was wondering (in another issue if desired) if you would be open to extending this open functionality, to also allow a custom domain?  for example
```shell
ws --open=local.myapp.com 
```

would open `local.myapp.com:<port>` in the browser.

Only thought is how to configure `open` when it can either be a string or boolean.  Let me know your thoughts either way!

**Comment by 75lb** [comment]
hi.. take a look at the current [lws master branch](https://github.com/lwsjs/lws), i pushed an update earlier on. 

The [current behaviour](https://github.com/lwsjs/lws/blob/master/lib/command/serve.js#L298) is to open either `127.0.0.1` or the hostname you passed to `--hostname`. So, to get your desired behaviour you'd run this command: 

```
ws --hostname=local.myapp.com --open
```

Is that what you need?

**Comment by 75lb** [comment]
i'm still testing the `opn` solution on all operating systems.. on macOs the call to `opn` doesn't resolve until you quit your browser, which is a known issue.. It make not be a blocker.. if it is, i'll try the `opener` library.

**Comment by thescientist13** [comment]
> Is that what you need?

@75lb 
That worked perfectly, thank you!

And yes, [this](https://github.com/lwsjs/lws/blob/master/lib/command/serve.js#L303) looks like it would get the best of both worlds!  🏆

**Comment by 75lb** [comment]
released in local-web-server v2.6.0, thanks.

**Comment by thescientist13** [comment]
Thank you!  Just upgraded and it is working great.  👍
